.buildkite/pipeline.yml

```diff
@@ -216,10 +216,21 @@ steps:
       queue: central
       slurm_ntasks: 1
 
-  - label: ":computer: :fast_forward: Bickley jet DG"
+  - label: ":computer: :fast_forward: Bickley jet DG core vs reference"
     key: "cpu_benchmark_bickleyjet_dg"
     command:
-      - "julia --color=yes --project=benchmarks/bickleyjet benchmarks/bickleyjet/runbenchmarks.jl"
+      - "julia --color=yes --project=benchmarks/bickleyjet benchmarks/bickleyjet/core_vs_ref.jl"
+    artifact_paths:
+      - "benchmarks/bickleyjet/*.png"
+    agents:
+      config: cpu
+      queue: central
+      slurm_ntasks: 1
```

Comment:
any slurm job cli flag  setting is passed through with the `slum_` prefix.  underscores get converted into hyphens so we don't have to escape, ex: `--mem-per-gpu=val` is `slum_mem_per_gpu: val` 

you'll need to adjust the number of tasks, or if you want to reserve a full node for perf testing

Reply:
Should I also add a `--exclusive`?

Reply:
I guess I did that in the past by reserving the whole node, but you can do that with `slurm_exlusive:  ""`